tests/test_indexer.py

```diff
@@ -159,6 +162,21 @@ def test_es_client_reuse(self):
 
         self.assertIs(es_client_after_first_call, es_client_after_second_call)
 
+    def test_subscription_notification(self):
+        bundle_key = self.load_test_data_bundle_for_path('fixtures/smartseq2/paired_ends')
+        sample_s3_event = self.create_sample_s3_bundle_created_event(bundle_key)
+        process_new_indexable_object(sample_s3_event, logger)
+
+        ElasticsearchClient.get(logger).indices.create(DSS_ELASTICSEARCH_SUBSCRIPTION_INDEX_NAME)
+        subscribe_for_notification(smartseq2_paired_ends_query,
+                                   "https://example.com/notification",
+                                   "6112f2a3-8b89-4e54-bbc0-65a98bf8fb8b")
+
+        bundle_key = self.load_test_data_bundle_for_path('fixtures/smartseq2/paired_ends')
+        sample_s3_event = self.create_sample_s3_bundle_created_event(bundle_key)
+        process_new_indexable_object(sample_s3_event, logger)
+        # TODO (mbaumann) Verify notification
```

Comment:
Please add an issue to track this. Testing the notification end-to-end is an important test case.